packages/headless/src/app/store.ts

```diff
@@ -17,13 +17,15 @@ interface ConfigureStoreOptions<Reducers extends ReducersMapObject> {
   preloadedState?: StateFromReducersMapObject<Reducers>;
   middlewares?: Middleware[];
   thunkExtraArguments: ThunkExtraArguments;
+  name?: string;
```

Comment:
I think we can make `name` required, since it will either be specified by the developer, or the default value of `coveo-headless`.